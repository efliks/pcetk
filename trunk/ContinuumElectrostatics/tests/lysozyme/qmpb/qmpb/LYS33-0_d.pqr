../0/site_LYS_33_d.pqr