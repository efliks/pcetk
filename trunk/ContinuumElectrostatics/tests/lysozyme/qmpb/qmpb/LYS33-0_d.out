Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named LYS33-0_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (2.9, 24.917, 6.506) 101 1
Finer level:    (2.9, 24.917, 6.506) 101 0.5
Finest level:   (2.9, 24.917, 6.506) 101 0.25


Self energy of LYS33-0_d in three dielectric environment = 18.5544
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of LYS33-0_d with background charge set in three dielectric environment = -0.245592
(probably in kcal/mole)



Interaction energies of LYS33-0_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 5.36765e-05
0   1 : 1.42697e-06
1   0 : 0.000113028
1   1 : 0.000132758
2   0 : 0.000193755
2   1 : 5.3282e-06
3   0 : 0.000701044
3   1 : 0.000157108
3   2 : -0.000116075
3   3 : -0.000588115
4   0 : 0.00014186
4   1 : -0.000514824
5   0 : 2.82338e-05
5   1 : -0.000353988
6   0 : -0.000147348
6   1 : -0.000792254
7   0 : 66.2148
7   1 : 37.1087
8   0 : 0.000487858
8   1 : -0.00715939
9   0 : -3.25362e-05
9   1 : -0.00047354
10   0 : 0.000125144
10   1 : -0.00180329
11   0 : 3.49897e-06
11   1 : -0.000523198
12   0 : 4.61413e-06
12   1 : -0.000296149
13   0 : 4.9319e-06
13   1 : -0.00026722
14   0 : 0.000452279
14   1 : 2.97479e-05
15   0 : 0.000101645
15   1 : 1.49995e-06
16   0 : 5.48712e-06
16   1 : -0.000322328
17   0 : 0.000409689
17   1 : 6.22544e-06
18   0 : -1.2801e-05
18   1 : -0.000417898
Total runtime: 7.66 s user 0.05 s system 
